examples/gno.land/p/jaekwon/book/book.gno

```diff
@@ -0,0 +1,379 @@
+package book
+
+/*
+
+GOALS:
+
+1. Create a Book (aka List) of text,
+   that is owned by a Book of Persons.
+
+2. Allow a Book of Persons to split/fork.
+
+*/
+
+// XXX open questions
+// * Do we want a directory?
```

Comment:
Yes, we need an equivalent of `std.Address` specifically for objects. The directory will function as the object catalog, similar to how the tm2 account module handles key-based accounts.